app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# :nodoc:
+class Comment < ApplicationRecord
```

Comment:
A comment also `belongs_to :user`. We should be able to grab a comment and execute `comment.user` to retrieve the user record that created the comment; however, we are not able to do that at the moment:

```
=> #<Comment id: 2, body: "ASDFG", post_id: 1, created_at: "2019-10-01 00:28:36", updated_at: "2019-10-01 00:28:36">
irb(main):008:0> c.user
Traceback (most recent call last):
        1: from (irb):8
NoMethodError (undefined method `user' for #<Comment:0x000055a45f4e0918>)
```

It should work the same way as it does for a Post:

```
irb(main):010:0> p = Post.first
  Post Load (0.6ms)  SELECT  "posts".* FROM "posts" ORDER BY "posts"."created_at" DESC LIMIT $1  [["LIMIT", 1]]
=> #<Post id: 1, description: "Hello World!", user_id: 1, created_at: "2019-10-01 00:27:53", updated_at: "2019-10-01 00:27:53">
irb(main):011:0> p.user
  User Load (0.6ms)  SELECT  "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 1], ["LIMIT", 1]]
=> #<User id: 1, username: "adriaanbd", email: "adriaanbd@gmail.com", created_at: "2019-10-01 00:27:30", updated_at: "2019-10-01 00:27:30", avatar_file_name: nil, avatar_content_type: nil, avatar_file_size: nil, avatar_updated_at: nil>
```

[`REQUIRED`]